app/models/cohort.rb

```diff
@@ -22,10 +21,6 @@ def self.active_registration_cohort
     where(registration_start_date: ..Date.current).order(start_year: :desc).first
   end
 
-  def self.active_npq_registration_cohort
-    where(npq_registration_start_date: ..Date.current).order(start_year: :desc).first.presence || current
```

Comment:
Will we drop `npq_registration_start_date` from `cohorts` table as well?

Reply:
We're keeping the columns for now, just removing the logic.

Reply:
I think maybe removing the column might be a good shout, let's discuss tomorrow